modules/auxiliary/scanner/ssl/ssl_version.rb

```diff
@@ -117,7 +117,7 @@ def get_metasploit_ssl_versions
       return Array.new(OpenSSL::SSL::SSLContext.new.ciphers.length) { |i| (OpenSSL::SSL::SSLContext.new.ciphers[i][1]).to_s }.uniq.reverse
     end
 
-    datastore['SSLVersion']
+    [datastore['SSLVersion']]
```

Comment:
Just some sanity questions, haven't had a deep look here:

- Should we have a split or anything here, i.e. `datastore['SSLVersion'].split(',').map(&:strip).reject(&:empty?)` if the user wants to specify multiple values
- Should these user provided values validated against `OpenSSL::SSL::SSLContext` in any capacity?

Reply:
1. I had never intended that way, I think most people would use this in a `sslscan` type way which would be all. if you were hunting for a vuln server, that vuln would likely just be against a single protocol, and so you'd target that.  I'm sure there are other use cases, but those were the two I originally envisioned.
2. Since the sslversion enum is generated based on the availability for the cipher in openssl+ruby+metasploit, the user input is validated:
```
msf6 auxiliary(scanner/ssl/ssl_version) > set sslversion sslv1
[-] The following options failed to validate: Value 'sslv1' is not valid for option 'SSLVersion'.
sslversion => All
```